src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala

```diff
@@ -31,4 +31,9 @@ object Configuration {
 
   lazy val MONGO_HOST: String = "mongodb://localhost:27017"
   lazy val WAIT_QUEUE_SIZE_MONGO: Int = 100
+
+  lazy val PATH_TO_DATASET_TO_SPLIT: String = appConf.getString("file.path_to_split_dataset")
+  lazy val SPLIT_BASE_DIR: String = appConf.getString("file.split_dataset_base_folder")
+  lazy val SPLIT_DATASET_BASE_FOLDER: String =  s"/home/${System.getProperty("user.name")}/$SPLIT_BASE_DIR"
```

Comment:
/home/${System.getProperty("user.name")}
There is already predefined system property in jvm - user.home, you can utilize it:
```
jshell> System.getProperty("user.home")
$1 ==> "/home/wsa-230a"
```
It returns environment specific path so for Windowns you'll get something like `C:\Users\wsa-230`

Reply:
rewrited